app/models/user.rb

```diff
@@ -470,6 +476,15 @@ def find_ids(options)
 
   protected
 
+  def promote_to_first_admin
+    account.update({first_admin_id: id})
+  end
+
+  def update_next_account_first_admin
```

Comment:
Can we call it `set_other_first_admin_to_account`?

Reply:
Or maybe just remove `next_`: `update_account_first_admin`

Reply:
I am not sure if the word `other` is appropriate though 🤔 because it sounds like a random other 🤔

Reply:
`Or maybe just remove next_: update_account_first_admin` haha ok 👍